README.md

```diff
@@ -143,6 +143,23 @@ You'll need apache2-utils on the machine where you plan to create the htpasswd f
 
 If you need to configure Nginx beyond what is possible using environment variables, you can provide custom configuration files on either a proxy-wide or per-`VIRTUAL_HOST` basis.
 
+#### Overriding default proxy settings
+
+If you want to override the default proxy settings for the nginx container, add a configuration file at `/etc/nginx/proxy_settings.conf`. A file with the default settings would
+look like this:
+
+```/etc/nginx/proxy_settings.conf
```

Comment:
Honestly I just put that there to mimic the block a little later in the file, assuming that it was done that way for a reason.  Sounds like they are both broken =]